### DISCUSSION TITLE
[Request] Implements SeerrBridge

### DISCUSSION THREAD

**Discussion by KnuffigerMaxi** [ahembree/ansible-hms-docker#108] [Ideas]
https://www.reddit.com/r/Overseerr/s/rPEc6iaWlB
https://github.com/Woahai321/SeerrBridge?tab=readme-ov-file#readme
https://github.com/debridmediamanager/debrid-media-manager

**Comment by ahembree** [comment]
Due to the various requirements to implement this (for both Seerrbridge and DMM), I don't believe it'd be a good fit for this project at this time. If there is a lot more interest in this from others then I might investigate further.
